### PR TITLE
Databox stable/2019-09-01 API Version

### PR DESCRIPTION
**Pull request by sashanm**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/databox/resource-manager/Microsoft.DataBox/stable/2019-09-01/databox.json [hunk 1]
```diff
@@ -0,0 +1,4160 @@
...
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "CopyStatus",
+            "modelAsExtensible": true,
+            "values": [
+              {
+                "value": "NotStarted",
+                "description": "Data copy hasnt started yet."
```
Comment (nschonni) [inline specification/databox/resource-manager/Microsoft.DataBox/stable/2019-09-01/databox.json]:
```suggestion
                "description": "Data copy hasn't started yet."
```

File: specification/databox/resource-manager/Microsoft.DataBox/stable/2019-09-01/databox.json [hunk 2]
```diff
@@ -0,0 +1,4160 @@
...
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ScheduleAvailabilityRequest"
+        }
+      ],
+      "properties": {
+        "expectedDataSizeInTeraBytes": {
+          "format": "int32",
+          "description": "The expected size of the data, which needs to be transfered in this job, in tera bytes.",
```
Comment (nschonni) [inline specification/databox/resource-manager/Microsoft.DataBox/stable/2019-09-01/databox.json]:
```suggestion
          "description": "The expected size of the data, which needs to be transferred in this job, in terabytes.",
```

File: specification/databox/resource-manager/Microsoft.DataBox/stable/2019-09-01/databox.json [hunk 3]
```diff
@@ -0,0 +1,4160 @@
...
+        "skuName"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ScheduleAvailabilityRequest"
+        }
+      ],
+      "properties": {
+        "expectedDataSizeInTeraBytes": {
```
Comment (nschonni) [inline specification/databox/resource-manager/Microsoft.DataBox/stable/2019-09-01/databox.json]:
```suggestion
        "expectedDataSizeInTerabytes": {
```

File: specification/databox/resource-manager/Microsoft.DataBox/stable/2019-09-01/databox.json [hunk 4]
```diff
@@ -0,0 +1,4160 @@
...
+        "contactDetails",
+        "shippingAddress",
+        "destinationAccountDetails",
+        "jobDetailsType"
+      ],
+      "type": "object",
+      "properties": {
+        "expectedDataSizeInTeraBytes": {
+          "format": "int32",
+          "description": "The expected size of the data, which needs to be transfered in this job, in tera bytes.",
```
Comment (nschonni) [inline specification/databox/resource-manager/Microsoft.DataBox/stable/2019-09-01/databox.json]:
```suggestion
          "description": "The expected size of the data, which needs to be transferred in this job, in terabytes.",
```

File: specification/databox/resource-manager/Microsoft.DataBox/stable/2019-09-01/databox.json [hunk 5]
```diff
@@ -0,0 +1,4160 @@
...
+      "description": "Job details.",
+      "required": [
+        "contactDetails",
+        "shippingAddress",
+        "destinationAccountDetails",
+        "jobDetailsType"
+      ],
+      "type": "object",
+      "properties": {
+        "expectedDataSizeInTeraBytes": {
```
Comment (nschonni) [inline specification/databox/resource-manager/Microsoft.DataBox/stable/2019-09-01/databox.json]:
```suggestion
        "expectedDataSizeInTerabytes": {
```

File: specification/databox/resource-manager/Microsoft.DataBox/stable/2019-09-01/databox.json [hunk 6]
```diff
@@ -0,0 +1,4160 @@
...
+          "description": "Location of the service.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "DiskScheduleAvailabilityRequest": {
+      "description": "Request body to get the availability for scheduling disk orders.",
+      "required": [
+        "expectedDataSizeInTeraBytes",
```
Comment (nschonni) [inline specification/databox/resource-manager/Microsoft.DataBox/stable/2019-09-01/databox.json]:
```suggestion
        "expectedDataSizeInTerabytes",
```

File: specification/databox/resource-manager/Microsoft.DataBox/stable/2019-09-01/databox.json [hunk 7]
```diff
@@ -0,0 +1,4160 @@
...
+          "$ref": "#/definitions/ShippingAddress",
+          "description": "Shipping address of the customer."
+        },
+        "deliveryPackage": {
+          "$ref": "#/definitions/PackageShippingDetails",
+          "description": "Delivery package shipping details.",
+          "readOnly": true
+        },
+        "returnPackage": {
+          "$ref": "#/def
```
Comment (nschonni) [inline specification/databox/resource-manager/Microsoft.DataBox/stable/2019-09-01/databox.json]:
```suggestion
                "description": "Identify request and response of active job limit for SKU availability."
```

**Comment by ravbhatnagar** [comment]
Signing off from ARM side.

**Comment by yungezz** [comment]
HI @sashanm  is this PR ready to merge?

**Comment by sashanm** [comment]
@yungezz  Yes. It is ready to merge

**Review by yungezz — APPROVED** [review]

**Pull request merged**